### ISSUE TITLE
Bulk API Image Type will be replace by secound entry

### ISSUE THREAD

**Issue by frostblogNet** [magento/magento2#24462]
As a merchant, I want to have seperate images with type thumbnail, small and image. 
This can be set using the admin backend, but it is not working from the rest api.

### Preconditions (*)

1. Magento Commerce/OpenSource 2.3.2
2.

### Steps to reproduce (*)

1. Create an Simple Product
2. Upload 2 image using the rest bulk api like this

```javascript
URL: {{base_url}}/rest/all/async/bulk/V1/products/bySku/media
Body:
[
  {
    "entry": {
      "media_type": "image",
      "label": "Image",
      "disabled": false,
      "position": 1,
        "types": [
            "image",
            "small_image"
        ],
      "content": {
        "base64EncodedData": "iVBORw0KGgoAAAANSUhEUgAAA+gAAAPoCAYAAABNo9TkAAAUvUlEQVR42u3XMQEAMAjAsOFozvHEgw1EcPAkEvo1fnY9AAAA4FQYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAg27QAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKAbdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoBh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKDLAAAAAAYdAAAADLpBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADLpBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAINu0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADLoMAAAAYNABAADAoBt0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDbtABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoBt0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoMsAAAAABh0AAAAMukEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoBh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMukEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAg27QAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAALAzXuRkvxT3x8QAAAAASUVORK5CYII=",
        "type": "image/png",
        "name": "20750.jpg"
      }
    },
    "sku": "49128"
  },
  {
    "entry": {
      "media_type": "image",
      "label": "Image",
      "disabled": false,
      "position": 8,
      "types": ["thumbnail"],
      "content": {
        "base64EncodedData": "iVBORw0KGgoAAAANSUhEUgAAA+gAAAPoCAYAAABNo9TkAAAUvUlEQVR42u3XMQEAMAjAsOFozvHEgw1EcPAkEvo1fnY9AAAA4FQYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAg27QAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKAbdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoBh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKDLAAAAAAYdAAAADLpBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADLpBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAIMOAAAAGHQAAAAw6AAAAIBBBwAAAINu0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADDoAAABg0AEAAMCgAwAAAAYdAAAADLoMAAAAYNABAADAoBt0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDbtABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoBt0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgAAACAQQcAAACDDgAAABh0AAAAMOgGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoAMAAAAGHQAAAAw6AAAAYNABAADAoMsAAAAABh0AAAAMukEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoBh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMukEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAgw4AAAAYdAAAADDoAAAAgEEHAAAAg27QAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAAGDQAQAAwKADAAAABh0AAAAMOgAAALAzXuRkvxT3x8QAAAAASUVORK5CYII=",
        "type": "image/png",
        "name": "20750_back.jpg"
      }
    },
    "sku": "49128"
  }
]
```
### Expected result (*)

1. In the backend there should be 2 images both with types.
2. The first image should be marked aus base,small and the secound should be thumbnail.

### Actual result (*)

1. Depending which image will be processed last only this types will be set, as you can see in the screenshot below.
![grafik](https://user-images.githubusercontent.com/19548641/64334264-7687af00-cfd8-11e9-8188-4d6fad8c7763.png)

**Comment by engcom-Bravo** [comment]
Hello @frostblogNet. Probably this issue has been fixed in the develop version. Here is the result we've got while trying to reproduce Your issue:
![api](https://user-images.githubusercontent.com/51680745/64425649-fb47fb00-d0b4-11e9-8a87-6de32eda551c.png)
Would You please try to reproduce the issue on 2.3-develop version too. Thank You.

**Comment by engcom-Bravo** [comment]
@frostblogNet, the contributor guide suggests that tickets that have not been active for two weeks should be closed. Please reopen as needed.

**Comment by frostblogNet** [comment]
Dear @engcom-Bravo 

sorry for the late response. I have´t tried 2.3-develop yet but this issue seems to be 

[2019-10-07 13:03:11] report.ERROR: The product can't be saved. [] []

We will try 2.3-develop next. I will come back if this is still an issue

**Comment by engcom-Bravo** [comment]
Hi @frostblogNet. Ok, but please when trying to reproduce the issue be sure to do it on a vanilla instance, e.i a fresh installation free of any third party extensions, custom code, etc.

**Comment by frostblogNet** [comment]
I can confirm that this issue is fixed in 2.3.3.
